crys3d/run_tests.py

```diff
@@ -3,16 +3,55 @@
 from libtbx import test_utils
 import libtbx.load_env
 
-tst_list = [
-"$D/regression/tst_hklinfo.py",
-#"$D/regression/tst_HKLviewer.py",
-]
+
+if "darwin" in sys.platform:
```

Comment:
You can also build up the test lists instead of copying
```python
tst_list = [ "$D/regression/tst_hklinfo.py" ]
tst_list_expected_unstable = [
   # fails sometimes due to websocket connection problem to webbrowser
   "$D/regression/tst_websocket.py",
   "$D/regression/tst_HKLviewerOSbrowserSliceK-9.py",
   "$D/regression/tst_HKLviewerOSbrowserBinFSigF.py",
]
other_tests = [
  "$D/regression/tst_HKLviewerQtGuiSliceK-9.py",
  "$D/regression/tst_HKLviewerQtGuiBinFSigF.py"
]
if sys.platform == "darwin" or sys.platform == "win32":
  tst_list.extend(other_tests)
else:
  tst_list_expected_unstable.extend(other_tests)
```

Reply:
done